src/node_messaging.cc

```diff
@@ -1645,6 +1645,31 @@ static void BroadcastChannel(const FunctionCallbackInfo<Value>& args) {
   }
 }
 
+static void ExposeLazyDOMExceptionPropertyGetter(
+    v8::Local<v8::Name> name, const v8::PropertyCallbackInfo<v8::Value>& info) {
+  auto context = info.GetIsolate()->GetCurrentContext();
+  Local<Function> domexception = GetDOMException(context).ToLocalChecked();
+  info.GetReturnValue().Set(domexception);
+}
+static void ExposeLazyDOMExceptionProperty(
+    const FunctionCallbackInfo<Value>& args) {
+  CHECK_GE(args.Length(), 1);  // target
+  CHECK(
+      args[0]
+          ->IsObject());  // target: Object where to define the lazy properties.
+
+  Isolate* isolate = args.GetIsolate();
+  auto target = args[0].As<v8::Object>();
+
+  target
+      ->SetLazyDataProperty(isolate->GetCurrentContext(),
+                            FIXED_ONE_BYTE_STRING(isolate, "DOMException"),
+                            ExposeLazyDOMExceptionPropertyGetter,
+                            Null(isolate),
+                            v8::DontEnum)
+      .Check();
```

Comment:
This really ought to avoid using `Check()` so that errors are appopriately propagated instead of crashing.